### DISCUSSION TITLE
[Implemented] OTA

### DISCUSSION THREAD

**Discussion by DJFlaxi** [philibertc/micronova_controller#36] [Ideas]
Hi philibertc,

I managed to update the D1 via OTA.

You have to install also the "ArduinoOTA" from the Library.

Can you include this in the code future sketch:

`
#include <ArduinoOTA.h>

void setup() {
  ArduinoOTA.setHostname("Pelletheater");
  ArduinoOTA.setPassword("micronova");
  ArduinoOTA.begin();
}

void loop() {
  ArduinoOTA.handle();
}
`

**Comment by philibertc** [comment]
Hello,

Great! I will incorporate this along with other simplifications into the program.
I will also make the hostname correspond to the base mqtt topic so that it can be easily found if you have several stoves.

Best Regards,
Philibert

**Reply by philibertc** [reply]
Hello,
It's now implemented!

Best Regards,
Philibert
